Source/Core/Core/Boot/Boot.h

```diff
@@ -151,3 +151,9 @@ struct StateFlags
 // Reads the state file from the NAND, then calls the passed update function to update the struct,
 // and finally writes the updated state file to the NAND.
 void UpdateStateFlags(std::function<void(StateFlags*)> update_function);
+
+/// Create title directories for the system menu (if needed).
+///
+/// Normally, this is automatically done by ES when the System Menu is installed,
+/// but we cannot rely on this because we don't require any system titles to be installed.
+void CreateSystemMenuTitleDirs();
```

Comment:
Yeah, I intended to make it a doxygen comment for the function. Makes it possible for IDEs and plugins to show a nice description :)

Reply:
IIRC VS also uses the two-slash ones, especially since there doesn't seem to be a C++ equivalent of C#'s doc-comments (like `<summary>` etc.).
Fine by me tho, just wanted to point it out.